.github/workflows/build-test.yml

```diff
@@ -217,6 +217,130 @@ jobs:
+          { component: components/locid, examples: "syntatically_canonicalize_locales filter_langids" },
+          { component: components/plurals, examples: "unread_emails elevator_floors" },
+          { component: components/uniset, examples: unicode_bmp_blocks_selector },
+          # Important! When adding new components, ensure that dhat is added as
+          # dev dependency, or else the job will fail.
+        ]
+
+    runs-on: ${{ matrix.os }}
+
+    # This job waits on the full build, even though it doesn't need anything from it.
+    # This way it is only triggered if the build is successful. The CI could be made
+    # a little bit faster by starting this job immediately, as it's fully independent.
+    needs: [build]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Nightly is only needed for the `--profile bench` line. Once that is stabilized,
+      # then this line is no longer needed.
+      - name: Install nightly toolchain
+        run: rustup toolchain install nightly
+
+      - name: Install cargo-make
+        uses: davidB/rust-cargo-make@v1
+        with:
+          version: latest
+
+      - name: Run the example with dhat-rs to collect memory information
+        run: |
+          cargo run --package memory_bench -- ${{ matrix.os }} ${{ matrix.example.component }} ${{ matrix.example.examples }}
+
+      # Benchmarking & dashboards job > (unmerged PR only) Convert benchmark output into
+      # dashboard HTML in a commit of a branch of the local repo.
+
+      - name: Store benchmark result & create dashboard (unmerged PR only)
+        # any action that is not a merge to master implies unfinished PR
+        if: github.event_name != 'push' || github.ref != 'refs/heads/master'
+        uses: gregtatum/github-action-benchmark@v1
+        with:
+          name: Heap – ${{ matrix.example.component }}
+          # The ndjson tool is only supported by the gregtatum fork of github-action-benchmark.
+          tool: 'ndjson'
+          benchmark-data-dir-path: ./benchmarks/memory/${{ matrix.os }}/${{ matrix.example.component }}
+          output-file-path: ./benchmarks/memory/${{ matrix.os }}/${{ matrix.example.component }}/output.ndjson
+          # Show alert with commit comment on detecting possible memory regression
+          alert-threshold: '200%'
+          fail-on-alert: true
+          # Requires one-time-only creation of this branch on remote repo. This will
+          # store the generated information.
+          gh-pages-branch: unmerged-pr-memory-data
+
+          # Do not store historical benchmark info of unfinished PRs. Commits seem to get
+          # made anyways, so make sure that the branch in `gh-pages-branch` is
+          # different from the branch used for merges to master/main branch.
+          auto-push: false
+
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-on-alert: true
+          alert-comment-cc-users: '@sffc,@zbraniecki,@echeran,@gregtatum'
```

Comment:
?

Reply:
I'm not sure what the question is exactly here, but I copied the alert settings from the other benchmarks, and added myself to the list. This area is part of the discussion I listed in the main area:

> How should the alerts be handled? Currently I kept the same defaults as the performance ones, but does this make sense?